README.adoc

```diff
@@ -167,6 +167,31 @@ Where :
 
 Beware : you will have concurrency issues if multiple containers are running on this single volume.
 
+=== Running integration tests against your James container
+
+We wrote some MPT (James' Mail Protocols Tests subproject) integration tests to validate a James 
+deployement.
+
+It uses the External-James module, that can easily use environment variables to locate a distant 
+IMAP server and run integration tests against it.
+
+For that, you need to handle the domain +domain+ and configured user +imapuser+ with password 
```

Comment:
"you need" ? I think you are talking about the james server.
It's more appropriate to say that "The target James Server needs to be configured with a domain ...."